src/content/docs/dev/send-tokens/sui/intro.mdx

```diff
@@ -332,39 +423,120 @@ The function takes four parameters:
+
+1. `self`: The module that will be updated once the `operator` is set.
+1. `operator`: The address of the `operator` to be set. The address that is added should be an [address](https://docs.sui.io/references/sui-api/sui-graphql/reference/types/objects/address) type of your Channel.
+
+It is defined as follows:
+
+```rust
+public fun add_operator<T>(self: &mut CoinManagement<T>, operator: address) {
+    self.operator.fill(operator);
+}
+```
+
+The `add_operator()` function must be called before the token is [registered](#register-coin). This function can be called by the address that has created the [Coin Management](#coin-management) object. However, once the coin is registered with ITS this function is no longer callable as ownership of the `Coin Management` object (which in turn owns the `Operator` object) is passed to ITS itself. If in the future you wish to change the `Operator` that was set before the coin was registered, you can transfer the `Operator` with the [transfer operator](#transfer-operator) function.
+
+#### Transfer Operator
+
+The `transfer_operatorship()` function takes four parameters:
+
+1. `self`: The module that will be updated once the new `operator` is set.
+1. `channel`: The channel that is able to update the `operator`.
+1. `token_id`: The id of the token that will receive the new `operator`.
+1. `new_operator`: The address of the new `operator` to be set. 
+
+
+```rust
+public fun transfer_operatorship<T>(
+    self: &mut InterchainTokenService,
+    channel: &Channel,
+    token_id: TokenId,
+    new_operator: Option<address>,
+) {}
+```
 
-An address authorized to update [flow limits](#flow-limit) for token transfers. When setting a new flow limit, the module verifies that the caller's channel address matches the stored operator address to ensure only the operator can make that change.
 
 ## Flow Limit
 
-Flow Limits represent the volume of a coin that can be transferred in/out of Sui via ITS. This limit plays a critical role in maintaining network integrity and security. When coins are sent out of Sui. The flow limit logic can be found in its own [Flow Limit Module](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/types/flow_limit.move)
+A [Flow Limit](/dev/send-tokens/interchain-tokens/flow-limit/) represent the volume of a coin that can be transferred in/out of Sui via ITS. This limit plays a critical role in maintaining network integrity and security. When coins are sent out of Sui. The flow limit logic can be found in its own [Flow Limit Module](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/types/flow_limit.move). The duration of a `flow_limit` is set to an `epoch`. An `epoch` lasts for six hours. It is calculated as follows:
```

Comment:
> ~~A [Flow Limit](/dev/send-tokens/interchain-tokens/flow-limit/) represent the volume of a coin that can be transferred in/out of Sui via ITS.~~

A [Flow Limit](/dev/send-tokens/interchain-tokens/flow-limit/) represents the volume of a coin that can be transferred in/out of Sui via ITS.